roles/registrator/templates/registrator.service.j2

```diff
@@ -7,12 +7,14 @@ Requires=docker.service
 Restart=on-failure
 TimeoutStartSec=0
 EnvironmentFile=/etc/environment
+Environment="DOCKER_HOST={{ registrator_docker_socket }}"
+
 ExecStartPre=-/usr/bin/docker kill registrator
 ExecStartPre=-/usr/bin/docker rm registrator
 ExecStartPre=/usr/bin/docker pull {{ registrator_image }}
 
 ExecStart=/usr/bin/bash -c "/usr/bin/docker run --rm --name registrator \
--v "{{ registrator_docker_socket }}:/tmp/docker.sock" \
+-v /var/run/weave/weave.sock:/tmp/docker.sock \
```

Comment:
Not really we just actually need better naming
registrator_docker_socket is the actual  socket address ->  = docker_host = unix://var/blah-blah ( used for environment variable)
What I want here is the path -> /var/blah-blah